fastapp/fastapp/ui/index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en" hidden>
+  <head>
+    <script src="https://use.fontawesome.com/3a2eaf6206.js"></script>
+    <script type="module" src="https://cdn.skypack.dev/twind/shim"></script>
+  </head>
+  <body class="bg-gray-100">
+    <div class="flex items-center justify-center h-screen">
+      <div class="overflow-hidden rounded-lg bg-white shadow w-96">
+        <div class="border-b border-gray-200 bg-white px-4 py-4 sm:px-6">
+          <h3 class="text-lg font-medium leading-6 text-gray-900">
+            Upload Hyperpack
+          </h3>
+        </div>
+
+        <div class="px-4 py-4">
+          <form
```

Comment:
Found a bug where the file selection is cached when navigating back to the upload page:


https://user-images.githubusercontent.com/9532762/189967091-3d2c6b7b-fd9c-496a-8e2f-c9009cc4eb14.mov



It's looking like adding `autocomplete="off"` to the `<form>` fixes this:


https://user-images.githubusercontent.com/9532762/189967524-a6dec3fc-17d4-4b42-8025-a1fceee5d18a.mov

Reply:
Good catch, added autocomplete off to clear out the form on back navigation